tests/integration/test_integration.py

```diff
@@ -25,10 +28,13 @@ def data_generator(num_docs, numpy_image_uri):
         yield doc
 
 
-def test_use_in_flow(numpy_image_uri):
+@pytest.mark.parametrize('dtype', ['numpy.uint8', 'numpy.float32', 'numpy.float64'])
+def test_use_in_flow(numpy_image_uri, dtype):
+    os.environ['DTYPE'] = dtype
```

Comment:
this should be a fixture that is unset at the end

Reply:
Cab you pass a pytest.mark.parametrize parameter to a fixture?

Reply:
yes, with `indirect`

Reply:
done